src/ered_cluster.erl

```diff
@@ -299,6 +306,9 @@ handle_info({slot_info, Version, Response, FromAddr}, State) ->
                     Nodes = ered_lib:slotmap_all_nodes(NewMap),
                     MasterNodes = new_set(ered_lib:slotmap_master_nodes(NewMap)),
 
+                    %% Open new clients or reactivate any not yet stopped.
+                    State1 = start_clients(Nodes, State),
+
                     %% Remove nodes if they are not in the new map.
                     Remove = maps:keys(maps:without(Nodes, State#st.nodes)),
 
```

Comment:
Let's use `State1` instead of `State` in all lines below this one? It may not matter, but it looks like a mistake to still use State.